app/src/main/java/org/eyeseetea/malariacare/DashboardActivity.java

```diff
@@ -40,28 +52,27 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.fragment_dashboard);
 
-//        if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
-//            // If the screen is now in landscape mode, we can show the dialog in-line so we don't need this activity
-//            finish();
-//            return;
-//        }
-
         if (savedInstanceState == null) {
-            DashboardUnsentFragment detailsFragment = new DashboardUnsentFragment();
-            detailsFragment.setArguments(getIntent().getExtras());
-            FragmentTransaction ft = getFragmentManager().beginTransaction();
-            ft.add(R.id.dashboard_details_container, detailsFragment);
-            ft.setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE);
-            ft.commit();
-            DashboardSentFragment completedFragment = new DashboardSentFragment();
-            detailsFragment.setArguments(getIntent().getExtras());
-            FragmentTransaction ftr = getFragmentManager().beginTransaction();
-            ftr.add(R.id.dashboard_completed_container, completedFragment);
-            ftr.setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE);
-            ftr.commit();
+            prepareUI();
```

Comment:
@ivanarrizabalaga  why do you extract this bit of code into a method? Is it not better to leave it as it was before the fork?